### PR TITLE
Fix spelling issues

### PR DESCRIPTION
**Pull request by nnsW3**
Hello
While looking through your docs I found several spelling issues.
Br, Elias,.

---

## PR-Codex overview
This PR updates comments in `DeploylessUniversalSigValidator.sol` to clarify the order of operations for signature validation based on EIP standards.

### Detailed summary
- Updated a comment to correct a typo from "striclty" to "strictly"
- Clarified the defined order of operations for signature validation based on EIP-6492

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/pages/experimental/eip5792/writeContracts.mdx
Comment (tmm) [inline site/pages/experimental/eip5792/writeContracts.mdx:1]: revert this change

File: site/pages/docs/actions/public/watchEvent.md
Comment (tmm) [inline site/pages/docs/actions/public/watchEvent.md:1]: revert this change

File: site/pages/docs/actions/public/verifyMessage.md
Comment (tmm) [inline site/pages/docs/actions/public/verifyMessage.md:1]: revert this change

**Comment by nnsW3** [comment]
done

**Pull request merged**